src/util/regexp/not-s-and-z.js

```diff
@@ -0,0 +1,24 @@
+define(function() {
+
+/**
+ * Unicode regular expression for: everything except symbols from categories S and Z
+ *
+ * Generated by:
+ *
+ * var s = regenerate()
+ *   .addRange( 0x0, 0x10FFFF )
+ *   .remove( require( "@unicode/unicode-8.0.0/General_Category/Math_Symbol/symbols" ) )
```

Comment:
The `not-s.js` has the following comment: _everything except math symbols, currency signs, dingbats, and box-drawing characters_. As far as I understood dingbats are not part of the `S` category, they belong to the _Other Number (No)_ (see https://www.compart.com/en/unicode/U+2776). Do you think this could be an issue?

Reply:
Thanks for flagging this. It could be an issue. Although I wrote that, all I remember is what's written 🙈 , let's remove and analyze the impact/difference.